### ISSUE TITLE
Wemos MQTT interface in OpenHS

### ISSUE THREAD

**Issue by openhsorg** [openhsorg/openhs#38]
Create OpenHS MQTT interface for Wemos (ESP-8266EX). https://www.wemos.cc/product/d1-mini-pro.html
Please consider general MQTT interface with any Wifi device communication via MQTT.

**Comment by fmikulu** [comment]
Implemented in dedicated branch 034228bc0c715d90a413a341b464e818d8cbe6c8

Don't forget to use proper config as described in:
https://github.com/openhsorg/openhs/blob/WmosUpd/modules/org.openhs.core.cfg/openhs/site/readme.md

There are other problems to be solved in:
https://github.com/openhsorg/openhs/blob/WmosUpd/hardware_units/u19_temperature_sensor_wemos_dht22/sketch_u19_temp_sensor/sketch_u19_temp_sensor.ino

Any ideas how to cope with it?

- hardcoded wifi ssid/pswd see lines 15,16:
```cpp
const char* ssid = "xxx";
const char* password = "xxx";
```
- hardcoded MQTT broker IP address, see line 35:
```cpp
mqtt.begin("192.168.1.26", net);
```
- hardcoded identification of the device itself, see lines 54, 55:
```cpp
"Addr\":\"192.168.1.26\"'
```

**Comment by openhsorg** [comment]
Coupling wifi sensor with router has specific Issue #39: https://github.com/openhsorg/openhs/issues/39

**Comment by fmikulu** [comment]
Merged 5c5c65c88839ced7f56bb30109ac3e0136cf663c
